README.md

```diff
@@ -279,7 +285,7 @@ docker-compose exec "web" bash -c "cd /var/www/mediawiki/extensions/WikibaseLexe
 
 While using PHP you can use remote xdebug debugging.
 
-To do so you need to set `IDELOCALHOST` in you local.env file to the IP of your local machine (where you run your IDE) as it appears to docker. Note with Docker for Mac, you can use `IDELOCALHOST=host.docker.internal`.
+To do so you need to set `IDELOCALHOST` in you local.env file to the IP of your local machine (where you run your IDE) as it appears to docker. Note with Docker for Mac, you can use `IDELOCALHOST=host.docker.internal`. When running on linux, you can set this to `auto`, which will cause the IP to be auto-detected when the container is created. When the IP changes, you can run `update-env` to update the environment. This will automatically restart the `web` service. 
```

Comment:
```suggestion
To do so you need to set `IDELOCALHOST` in you local.env file to the IP of your local machine (where you run your IDE) as it appears to docker. Note with Docker for Mac, you can use `IDELOCALHOST=host.docker.internal`. When running on Linux, you can set this to `auto`, which will cause the IP address to be auto-detected when the container is created. When the address changes, you can run `update-env` to update the environment. This will automatically restart the `web` service. 
```